python-packages/contract_wrappers/src/zero_ex/contract_wrappers/utils.py

```diff
@@ -0,0 +1,18 @@
+from eth_utils import remove_0x_prefix
+
+
+def match_order_maker_with_cancellor(maker_address, cancellor_address):
```

Comment:
Since this is so specific to cancelling orders, I think it should live in exchange_wrapper.py.  And further, I suggest we actually get rid of the method, and just have an `if` statement everywhere that was calling it; but instead of re-writing that exception message everywhere, how about we encapsulate it in a new exception class, maybe `exchange_wrapper.CancelDisallowedError(maker_addr, cancellor_addr)`, and have all the former call sites of this method just say `if addr1 != addr2: raise CancelDisallowedError(addr1, addr2)`.  What do you think?  If we decide to keep it as a helper method (instead of a custom exception) I'd like to see it renamed to something more descriptive, maybe `raise_if_wrong_cancellor()` or something, just so it's totally clear when reading the calls to it.

Reply:
done.